### ISSUE TITLE
Bookmarks Alt tags - more descriptive

### ISSUE THREAD

**Issue by adrianspeyer** [vanilla/vanilla#9360]
**Describe the Feature**
Bookmark alt tag should include the title of the discussion bookmarked.

**User Story**
If customer using screen reader, or using analytics tag manger, no easy way to know on bookmark button click which exact discussion is bookmarked. 

Spoke with @slafleche and he suggested I add this as a feature enhancement.

**Comment by slafleche** [comment]
Will require new translation strings:

`Unbookmark %s` and `Bookmark %s` and then just pass the discussion through. https://github.com/vanilla/vanilla/blob/master/applications/vanilla/views/discussions/helper_functions.php#L72

To my knowledge, we have no clients that actually use the visible text for the bookmarks, so this should be easy to push without worrying about themes.

We should do something similar with the comments. One of our client suggested we add something along the lines of "Joe Blow said: " in front of each comment, so they could be understood out of context. They churned before we got their official recommendation. Would be nice to get feedback from a client.
